### ISSUE TITLE
Elevated plasma palmitoleylcarnitine, C16:1

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5977]
**Preferred term label:** Elevated plasma palmitoleylcarnitine, C16:1

**Synonyms**

**Definition (free text, please give PubMed ID)** Elevated Palmitoleylcarnitine concentration in the blood.

**Parent term (use hpo.jax.org/app)** Abnormal circulating acetylcarnitine concentration HP:0012071

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
adding term
